Description: Don't run failing test.
 That way we can still benefit by running all the other tests.
Author: Jeremy Bicha <jbicha@ubuntu.com>
Bug: https://bugzilla.gnome.org/782215
---
 tests/conform/Makefile.am |    1 -
 1 file changed, 1 deletion(-)

--- a/tests/conform/Makefile.am
+++ b/tests/conform/Makefile.am
@@ -24,7 +24,6 @@ actor_tests = \
 	actor-offscreen-redirect \
 	actor-paint-opacity \
 	actor-pick \
-	actor-shader-effect \
 	actor-size \
 	$(NULL)
 
